gcc.libc.imp

```diff
@@ -221,7 +221,7 @@
   { "include": [ "<sys/dtrace.h>", "private", "<dtrace.h>", "public"] },
   { "include": [ "<sys/paths.h>", "private", "<paths.h>", "public"] },
   { "include": [ "<sys/syslimits.h>", "private", "<limits.h>", "public"] },
-  { "include": [ "<sys/ttycom.h>", "private", "<termios.h>", "public"] },
+  { "include": [ "<sys/ttycom.h>", "private", "<sys/ioctl.h>", "public"] },
```

Comment:
Might be worth adding a comment here that `sys/ttycom.h` is a FreeBSD libc header, so people don't clean it up after finding that the header doesn't exist on a Linux system.